src/lib.rs

```diff
@@ -538,12 +539,26 @@ pub fn filename_to_module(filename: &str) -> StdString {
     name.replace("/", ".")
 }
 
+fn get_env_path() -> Option<String> {
+    match env::var("GLUON_PATH") {
+        Ok(val) => Some(val),
+        Err(_) => None,
+    }
+}
+
 /// Creates a new virtual machine with support for importing other modules and with all primitives
 /// loaded.
 pub fn new_vm() -> RootedThread {
     let vm = RootedThread::new();
-    vm.get_macros().insert(String::from("import"),
-                           ::import::Import::new(::import::DefaultImporter));
+    let gluon_path = get_env_path().unwrap_or(String::from("."));
```

Comment:
No need for a separate function here. You can write this as `env::var("GLUON_PATH").unwrap_or(String::from("."))` as `Result` have the same [unwrap_or](https://doc.rust-lang.org/core/result/enum.Result.html#method.unwrap_or) method.